src/js/jenkins_loader.js

```diff
@@ -0,0 +1,97 @@
+ 'use strict;'
+
+var JenkinsLoader = function (url, jobName) {
```

Comment:
Using the same name for the file and the object that it defines makes the code a bit more maintainable.

Reply:
What's your suggestion? I had a look at https://github.com/betaflight/betaflight-configurator/blob/master/src/js/release_checker.js when developing this

Reply:
Ouch, good point. We do have a mix of snake_case and CamelCase for file names (and for object names). I normally go for file names that are identical to the object they define (`JenkinsLoader` in this case). I don't know what I thought when I named `release_checker.js`, I suspect I must have renamed the object during development and forgotten about the file name.

Reply:
I could do a rename-refactor I'm a separate PR for multiple files